esy.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "bs-platform",
+  "version": "5.2.0-dev.2",
+  "dependencies": {
+    "ocaml": "4.02.3000+BS"
+  },
+  "devDependencies": {
+    "ocaml": "4.02.3000+BS"
+  },
+  "resolutions": {
+    "ocaml": "ulrikstrid/ocaml:package.json#e433d5b3fbe8963ac98257d1ed3b32c01515e07d"
+  },
+  "esy": {
+    "buildsInSource": true,
+    "buildDev": [
```

Comment:
Should I remove this part as well since the ninja.js file is not working with the setup any more? @bobzhang